File: JavaCode/DialogTest/app/src/main/java/com/example/dialogtest/MainActivity.java
package com.example.dialogtest;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.app.Dialog;
import android.content.DialogInterface;
import android.icu.text.IDNA;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import java.security.PrivateKey;
import java.security.PublicKey;
import java.util.Timer;
import java.util.TimerTask;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button showButton = findViewById(R.id.ShowButton1);
        showButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ShowDialog();
            }
        });
    }
    public void ShowDialog(){
        //AlertDialog dialog = null;
        AlertDialog.Builder builder = new AlertDialog.Builder(this);
        builder.setTitle("确认对话框");
        builder.setMessage("Welcome!What's your name?");
        final View filename = new EditText(MainActivity.this);
        builder.setView(filename);
        builder.setPositiveButton("我叫....", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Toast.makeText(MainActivity.this,"Hi!欢迎加入我们",Toast.LENGTH_LONG).show();
            }
        });
        builder.setNegativeButton("不告诉你", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Toast.makeText(MainActivity.this,"那好吧...",Toast.LENGTH_LONG).show();
            }
        });
        builder.setCancelable(false);
        AlertDialog dialog = builder.create();
        dialog.show();

    }

}

File: KunkunReader/app/src/main/java/com/xz2dai/kunkunreader/MainActivity.kt
package com.xz2dai.kunkunreader

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.Window
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import java.util.concurrent.Executors

class MainActivity : AppCompatActivity(),View.OnClickListener {

    //线程池
    val threadPoolExecutor = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 3 + 2)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        supportRequestWindowFeature(Window.FEATURE_NO_TITLE) //取消标题栏

        Main_AddLocal.setOnClickListener(this)
        Main_Config.setOnClickListener(this)
        Main_ToInfo.setOnClickListener(this)
        Main_ToShop.setOnClickListener(this)
    }

    fun showToast(msg: CharSequence) {
        showToast(msg)
    }

    override fun onClick(v: View?) {
        when(v){
            Main_AddLocal -> {
                showToast("功能开发中")
            }
            Main_Config -> {
                showToast("功能开发中")
            }
            Main_ToInfo -> {
                startActivity(Intent(this,UserActivity::class.java))
            }
            Main_ToShop -> {
                startActivity(Intent(this,ShopActivity::class.java))
            }

        }
    }
}